src/shared/components/EmptyGraphError.scss

```diff
@@ -25,5 +27,6 @@
 }
 
 .empty-graph-error--scroll {
+  width: 90% !important;
```

Comment:
is it possible to apply this style without `!important`, by being more [specific?](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

For example, do

```css
div.empty-graph-error--scroll {
  width: 90%
}
```
or 

```css
.empty-graph-error--scroll pre {
  width: 90%
}
```

or

```css
.empty-graph-error--scroll pre code {
  width: 90%
}
```
work?

Reply:
I agree! and I tried those, I think the issue is that the DappersScrollBar component in clockface is overriding the styles.

One way around it is to pass `style` prop to the `DappersScrollBar` component, would that be preferred? 

<img width="548" alt="Screen Shot 2022-06-01 at 8 33 21 AM" src="https://user-images.githubusercontent.com/18511823/171443009-f16c178a-e4ea-4077-91a7-f23001bca3a9.png">